src/core_modules/capture-core/components/WidgetAssignee/DisplayMode.component.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+import React from 'react';
+import i18n from '@dhis2/d2-i18n';
+import { Button, spacers, UserAvatar } from '@dhis2/ui';
+import { withStyles } from '@material-ui/core/styles';
+import { ConditionalTooltip } from 'capture-core/components/Tooltips/ConditionalTooltip';
+import type { Assignee } from './WidgetAssignee.types';
+
+const styles = () => ({
+    wrapper: {
+        display: 'flex',
+        alignItems: 'center',
+    },
+    editButton: {
+        margin: spacers.dp4,
+    },
+    assignButton: {
+        margin: spacers.dp4,
+    },
+    avatar: {
+        margin: spacers.dp4,
+    },
+});
+
+type Props = {
+    assignee: Assignee | null,
+    onEdit: () => {},
+    writeAccess: boolean,
+    ...CssClasses,
+};
+
+const DisplayModePlain = ({ assignee, onEdit, writeAccess, classes }: Props) => (
+    assignee ? (
+        <div className={classes.wrapper}>
+            <div>
+                {i18n.t('Assigned to')}
+                <UserAvatar name={assignee.name} className={classes.avatar} />
```

Comment:
@simonadomnisoru No picture unless we add `avatarId`, could you look into that?
@cooper-joe  I'm inclined to think the widget would look better if we removed the "Assigned to" part, would that work you think?

Reply:
@superskip Nice catch! I added the `avatarId`. The `UserAvatar` UI component falls back to the name initials only if the user does not have an avatar configured. Can you have a look? Thanks!

Reply:
First time viewing an assignee after reloading the app there is a flash showing the initials before the picture is shown. Not ideal (in terms of quality), maybe something we should address while we're at it. Should I reach out to @cooper-joe about the "Assigned to" suggestion, or have you already done so @simonadomnisoru?

Reply:
@superskip I updated the CSS with the mockup properties and implemented an avatar loading check. Can you have a look? Thank you for the feedback!